rqt_image_view failing to build on ROS 2 build farm

ROS distro: Rolling
Version: 1.1.1-1
Platforms:
  * [Ubuntu AMD64](https://build.ros2.org/job/Rbin_uF64__rqt_image_view__ubuntu_focal_amd64__binary/)
  * [Ubuntu ARM64](https://build.ros2.org/job/Rbin_ufv8_uFv8__rqt_image_view__ubuntu_focal_arm64__binary/)
  * [RHEL AMD64](https://build.ros2.org/job/Rbin_rhel_el864__rqt_image_view__rhel_8_x86_64__binary/)

This package version was building previously it is probable that this build failure is the result of changes in dependencies.

```
[ 50%] Building CXX object CMakeFiles/rqt_image_view.dir/src/rqt_image_view/image_view.cpp.o
/usr/lib/ccache/c++  -DDEFAULT_RMW_IMPLEMENTATION=rmw_cyclonedds_cpp -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DRCUTILS_ENABLE_FAULT_INJECTION -Drqt_image_view_EXPORTS -I/tmp/binarydeb/ros-rolling-rqt-image-view-1.1.1/obj-x86_64-linux-gnu -I/tmp/binarydeb/ros-rolling-rqt-image-view-1.1.1 -I/tmp/binarydeb/ros-rolling-rqt-image-view-1.1.1/include -isystem /opt/ros/rolling/include -isystem /opt/ros/rolling/include/pluginlib -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -isystem /usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++ -isystem /usr/include/opencv4 -isystem /opt/ros/rolling/include/class_loader  -g -O2 -fdebug-prefix-map=/tmp/binarydeb/ros-rolling-rqt-image-view-1.1.1=. -fstack-protector-strong -Wformat -Werror=format-security -DNDEBUG -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC   -Wall -Wextra -Wpedantic -fPIC -std=gnu++17 -o CMakeFiles/rqt_image_view.dir/src/rqt_image_view/image_view.cpp.o -c /tmp/binarydeb/ros-rolling-rqt-image-view-1.1.1/src/rqt_image_view/image_view.cpp
In file included from /tmp/binarydeb/ros-rolling-rqt-image-view-1.1.1/src/rqt_image_view/image_view.cpp:33:
/tmp/binarydeb/ros-rolling-rqt-image-view-1.1.1/include/rqt_image_view/image_view.h:40:10: fatal error: image_transport/image_transport.hpp: No such file or directory
   40 | #include <image_transport/image_transport.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[4]: *** [CMakeFiles/rqt_image_view.dir/build.make:78: CMakeFiles/rqt_image_view.dir/src/rqt_image_view/image_view.cpp.o] Error 1
make[4]: Leaving directory '/tmp/binarydeb/ros-rolling-rqt-image-view-1.1.1/obj-x86_64-linux-gnu'
make[3]: *** [CMakeFiles/Makefile2:81: CMakeFiles/rqt_image_view.dir/all] Error 2
make[3]: Leaving directory '/tmp/binarydeb/ros-rolling-rqt-image-view-1.1.1/obj-x86_64-linux-gnu'
make[2]: *** [Makefile:144: all] Error 2
make[2]: Leaving directory '/tmp/binarydeb/ros-rolling-rqt-image-view-1.1.1/obj-x86_64-linux-gnu'
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -53,7 +53,6 @@ set(rqt_image_view_INCLUDE_DIRECTORIES
   ${rqt_gui_cpp_INCLUDE_DIRS}
   ${sensor_msgs_INCLUDE_DIRS}
   ${geometry_msgs_INCLUDE_DIRS}
-  ${cv_bridge_INCLUDE_DIRS}
   ${Qt5Widgets_INCLUDE_DIRS}
 )
 
@@ -79,7 +78,7 @@ target_link_libraries(
   image_transport::image_transport
   ${sensor_msgs_LIBRARIES}
   ${geometry_msgs_LIBRARIES}
-  ${cv_bridge_LIBRARIES}
+  cv_bridge::cv_bridge
   Qt5::Widgets
 )
 

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -51,7 +51,6 @@ set(rqt_image_view_INCLUDE_DIRECTORIES
   ${rclcpp_INCLUDE_DIRS}
   ${qt_gui_cpp_INCLUDE_DIRS}
   ${rqt_gui_cpp_INCLUDE_DIRS}
-  ${image_transport_INCLUDE_DIRS}
   ${sensor_msgs_INCLUDE_DIRS}
   ${geometry_msgs_INCLUDE_DIRS}
   ${cv_bridge_INCLUDE_DIRS}
@@ -77,7 +76,7 @@ target_link_libraries(
   ${rclcpp_LIBRARIES}
   ${qt_gui_cpp_LIBRARIES}
   ${rqt_gui_cpp_LIBRARIES}
-  ${image_transport_LIBRARIES}
+  image_transport::image_transport
   ${sensor_msgs_LIBRARIES}
   ${geometry_msgs_LIBRARIES}
   ${cv_bridge_LIBRARIES}

